packages/gatsby-source-drupal/README.md

```diff
@@ -182,6 +182,12 @@ module.exports = {
 }
 ```
 
+_NOTES_:
+
+When using [includes](https://www.drupal.org/docs/8/modules/jsonapi/includes) in your JSON:API calls the included data will be automatically become available to query, even if the link types are skipped using `disallowedLinkTypes`.
```

Comment:
```suggestion
When using [includes](https://www.drupal.org/docs/8/modules/jsonapi/includes) in your JSON:API calls to the included data will automatically become available to query, even if the link types are skipped using `disallowedLinkTypes`.
```
I think there is a word missing here? Let me know if this alters the intended meaning.

Reply:
```suggestion
When using [includes](https://www.drupal.org/docs/8/modules/jsonapi/includes) in your JSON:API calls the included data will automatically become available to query, even if the link types are skipped using `disallowedLinkTypes`.
```

Thanks for your suggestion! There's indeed a word that doesn't belong there (be), but the "to" isn't needed because JSON:API call is not to the data.